recipes-devtools/clang/clang_git.bb

```diff
@@ -55,13 +55,13 @@ def get_clang_target_arch(bb, d):
 def get_clang_experimental_target_arch(bb, d):
     return get_clang_experimental_arch(bb, d, 'TARGET_ARCH')
 
-PACKAGECONFIG ??= "compiler-rt libcplusplus shared-libs lldb-wchar \
+PACKAGECONFIG ??= "compiler-rt libcplusplus shared-libs lldb-wchar clangd \
```

Comment:
I'm not sure how sensible it is to enable `clangd` in the target build (and to also deploy it to sysroot during preprocessing) but I've added to remain compatible with the current behaviour.

Reply:
I think disabling it for target by default makes sense perhaps, however, disabling it for native is going to be a bit questionable, since clang-native is used as part of clang-cross as well, so I think having language server during cross builds is a fair ask. 

Can you try your builds against clang-18 pull request and see if this fixes the g++ ICE if it does then perhaps we can add this option and disable it only for target.

Reply:
It looks like the ICE is gone with Clang 18. I'll update the PR to also enable clangd by default in the `native` build (and disable for the target).

Reply:
Actuall I stand corrected and it is still triggering an ICE:
```
| ../../../../../work-shared/llvm-project-source-18.1.0-r0/git/clang-tools-extra/clangd/index/dex/dexp/Dexp.cpp: In substitution of ‘template<class _Functor, class, class> std::function<std::unique_ptr<clang::clangd::{anonymous}::
Command>()>::function(_Functor) [with _Functor = <missing>; <template-parameter-1-2> = <missing>; <template-parameter-1-3> = <missing>]’:
| ../../../../../work-shared/llvm-project-source-18.1.0-r0/git/clang-tools-extra/clangd/index/dex/dexp/Dexp.cpp:372:1:   required from here
| ../../../../../work-shared/llvm-project-source-18.1.0-r0/git/clang-tools-extra/clangd/index/dex/dexp/Dexp.cpp:372:1: internal compiler error: Segmentation fault
|   372 | };
|       | ^
| Please submit a full bug report,
| with preprocessed source if appropriate.
| See <file:///usr/share/doc/gcc-10/README.Bugs> for instructions.
```
Nevertheless having the PACKAGECONFIG option available to simply turn compilation off in our builds is fine for me.